packages/react-table/src/components/Table/base/types.tsx

```diff
@@ -163,6 +163,15 @@ export interface ThSelectType {
   isHeaderSelectDisabled?: boolean;
 }
 
+export interface ThCollapseType {
+  /** On toggling the expansion */
+  onToggle?: OnCollapse;
+  /** Whether all are expanded */
+  isCollapsed: boolean;
+  /** Flag indicating the collapse in the th is disabled */
+  isHeaderCollapseDisabled?: boolean;
```

Comment:
does disabling the Collapse button in the header work? I don't think we have a disabled expand/collapse button option for the other (non-header) rows?

I don't see it referenced in collapse.tsx or CollapseColumn, so I don't know how it'd be disabling the toggle? I might be missing something obvious

Reply:
ah, this was just me looking at some of the select stuff and hoping it all translated over to expand